spec/system/task_spec.rb

```diff
@@ -3,22 +3,22 @@
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
     context '正常系' do
+      let(:project) { FactoryBot.create(:project) }
```

Comment:
設定ファイルに追記することで`FactoryBot`の記載は省略できます。
```suggestion
      let(:project) { create(:project) }
```

Reply:
修正しました。

Reply:
eea4c16